src/templates/partials/layout.hbs

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=yes">
+    <meta name="format-detection" content="telephone=no, date=no, address=no, email=no, url=no">
+    <meta name="x-apple-disable-message-reformatting">
+    <meta name="color-scheme" content="light dark">
+    <meta name="supported-color-schemes" content="light dark">
+    <title>{{subject}}</title>
+    <!--[if mso]>
+    <noscript>
+      <xml>
+        <o:OfficeDocumentSettings>
+          <o:PixelsPerInch>96</o:PixelsPerInch>
+        </o:OfficeDocumentSettings>
+      </xml>
+    </noscript>
+    <![endif]-->
+    <style type="text/css">
+      :root {
+        color-scheme: light dark;
+        supported-color-schemes: light dark;
+      }
+    </style>
+  </head>
+  <body class="body" xml:lang="en">
+    {{#if preheader}}
+      <div style="display: none; max-height: 0px; overflow: hidden;">
+        {{preheader}}
+      </div>
+
+      <div style="display: none; max-height: 0px; overflow: hidden;">
+        &#847; &zwnj; &nbsp; &#8199; &#65279; &#847; &zwnj; &nbsp; &#8199; &#65279; &#847; &zwnj; &nbsp; &#8199; &#65279; &#847; &zwnj; &nbsp; &#8199; &#65279; &#847; &zwnj; &nbsp; &#8199; &#65279; &#847; &zwnj; &nbsp; &#8199; &#65279; &#847; &zwnj; &nbsp; &#8199; &#65279; &#847; &zwnj; &nbsp; &#8199; &#65279;
```

Comment:
Preheader: https://www.goodemailcode.com/email-code/preheader

Reply:
I see, isn't there a way to convert text to Preheader? So we can customize / see it it more easily

Reply:
preheader is currently defined in `templates/index.ts`, as a custom text.

Reply:
Ok so it's not hardcoded anymore, sorry missed that.

Reply:
preheader is defined at line 31, the lines you highlighted are related to the padding after the preheader, so that email engine does not fill too short preheader with email body content